spec/selective/ruby/core/controller_spec.rb

```diff
@@ -10,6 +10,7 @@
   before do
     allow(Process).to receive(:spawn).and_return(123)
     allow(controller).to receive(:handle_termination_signals)
+    allow(controller).to receive(:wait_for_connectivity)
```

Comment:
It's a tricky method to test because it contains a blocking read that'll screw up the test run if we let it. Open to ideas about how we can test it. It is tested implicitly by running the tests with selective so 🤷🏻